src/compat/eme/set_media_keys.ts

```diff
@@ -37,12 +35,15 @@ function _setMediaKeys(
   elt : HTMLMediaElement,
   mediaKeys : MediaKeys|ICustomMediaKeys|null
 ) : any {
-  if (mediaKeys === null) {
-    return;
+  if (customSetMediaKeys !== null) {
```

Comment:
The flow seems a little convoluted now.

Why not doing like for `requestMediaKeySystemAccess` and defining a single `setMediaKeys(mediaElement, mediaKeys)` function, which depending on the environment will call `mediaElement.setMediaKeys(mediaKeys)` or `mediaElement.webkitSetMediaKeys(mediaKeys)`, or even (for old webkits) something like `mediaKeys._videoElement = mediaElement` (as it seems to be implemented that way in old webkit's custom media keys where the video element is passed to the createSession equivalent) etc.

Here we jump through several compat functions which all have different syntaxes, it seems harder to understand.